test/stability-tests/src/it/java/software/amazon/awssdk/stability/tests/s3/S3AsyncStabilityTest.java

```diff
@@ -56,16 +55,29 @@ public void putObject_getObject() {
     }
 
     @RetryableTest(maxRetries = 3, retryableException = StabilityTestsRetryableException.class)
-    @Disabled // Test needs to be fixed so that it doesn't rely on static bucket/key
     public void getLargeObject() throws IOException {
         LOGGER.info(() -> "Starting to test getLargeObject");
-        verifyObjectExist(LARGE_KEY_BUCKET_NAME, LARGE_KEY_NAME, (long) 2e+9);
+        RandomTempFile file = null;
+        try {
+            file = new RandomTempFile((long) 2e+9);
```

Comment:
Would be more efficient just to stream random data on the fly, maybe using something like [`Flowable.repeat()`](http://reactivex.io/RxJava/2.x/javadoc/io/reactivex/Flowable.html#repeat-long-)?

Reply:
Yeah, but I figured we could test using `FileAsyncRequestBody` to upload large object this way.

Reply:
Hmm I guess that fine. I don't like conflating testing the two components this way, especially since the test is called **get**LargeObject though.  I'd prefer two distinct test cases if we wanted to explicitly test file provider.

Reply:
Sure, I'll rename the test to make it more clear. We'd need to run get and then put in the same test though, cause the latter depends on the former.